config/test_config.go

```diff
@@ -19,7 +19,7 @@ func SetTestDefaultConfig() {
 		"BRUM_PERSISTANCE_DATABASE_STRATEGY": "all_in_one_db",
 		"BRUM_PERSISTANCE_TABLE_STRATEGY":    "all_in_one_table",
 		"BRUM_BACKUP_ENABLED":                "false",
-		"BRUM_BACKUP_DIRECTORY":              "/home/basicrum_backup",
+		"BRUM_BACKUP_DIRECTORY":              "/home/basicrum_archive",
```

Comment:
@ivan-stankov-golang is it possible that we revert more than needed here? 

This change seem to be unrelated. Could you check the whole PR for similar things that slipped?